.github/workflows/e2e.yml

```diff
@@ -0,0 +1,188 @@
+name: e2e
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - "**"
+
+env:
+  K8S_VERSION: "v1.27.3"
+  KIND_VERSION: "v0.20.0"
```

Comment:
hmm, we are not going to release a new version, and the fix will be on main https://github.com/kubernetes-sigs/kind/pull/3438

Reply:
no worries, I think we can workaround this https://github.com/mneverov/cluster-cidr-controller/pull/20/files#r1417428203